atom/browser/api/atom_api_importer.cc

```diff
@@ -18,7 +18,7 @@
 #include "base/strings/utf_string_conversions.h"
 #include "base/threading/thread_task_runner_handle.h"
 #include "brave/browser/brave_content_browser_client.h"
-#include "chrome/browser/importer/external_process_importer_host.h"
+#include "brave/browser/importer/brave_external_process_importer_host.h"
```

Comment:
I think these changes are unintentional? If you can clean them up I'll get it merged